devtools-ci-index.yaml

```diff
@@ -3778,6 +3873,21 @@
             ee_test_start_time: '20 6,18 * * *'
             timeout: 60m
             zabbix_enabled: false
+        - 'devtools-prebooster-test-{test_url}-{cluster}-{booster_name}':
+            test_url: openshift.io
+            prebooster_test_start_time: '5 0-23/4 * * *'
+            osio_prebooster_creds_token: 'prebooster_token_1'
+            project_name: osio-ci-boost1-preview-stage
+            booster_name: golang-health-check
+            cluster: https://api.starter-us-east-2a.openshift.com
+            booster_repo: golang-starters
+            report_dir: target
+            ui_headless: true
+            zabbix_enabled: true
+            zabbix_server: zabbix.devshift.net
+            zabbix_port: 9443
+            zabbix_host: qa_openshift.io
+            zabbix_metric_prefix: "prebooster.{booster_name}"
```

Comment:
AFAIK this variable substitution doesn't work, you need to do it inside `shell` section

Reply:
This is working now for the BDD tests - so I guess it is working.